File: samples/JavaScript/Json Store/Jsql/Find Json.js
session.setAsync(false);

var brand = "Bally";

var sql = "SELECT P.$.name name"
	+" FROM pinball P"
	+" WHERE LOWER(P.$.brand)=LOWER(:brand)";

var array = session.openJsqlCursor().database("playground")
			.query(sql).param("brand",brand).findJson();

var pinballs = "";
for(var i=0; i<array.length; i++) {
	pinballs += "\n"+array[i]["name"];
}

darwino.Utils.setText("content","Pinballs={0}",pinballs);

File: samples/Java/Json Store/Jsql/Find Json.java
String brand = "Bally";

String sql = "SELECT P.$.name name"
	+" FROM pinball P"
	+" WHERE LOWER(P.$.brand)=LOWER(:brand)";

JsonArray array = (JsonArray)session.openJsqlCursor().database("playground")
			.query(sql).param("brand",brand).findJson();

String pinballs = "";
for(int i=0; i<array.size(); i++) {
	pinballs += "\n"+array.getObject(i).getString("name");
}

_formatText("Pinballs={0}",pinballs);
